packages/pluggableWidgets/datagrid-web/tests/e2e/specs/DataGrid.spec.ts

```diff
@@ -0,0 +1,93 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+
+describe("datagrid-web", () => {
+    beforeAll(() => {
+        page.open(); // resets page
+    });
+
+    describe("capabilities: sorting", () => {
+        it("changes order of data to ASC when clicking sort option", () => {
+            const datagrid = page.getWidget("datagrid1");
+            datagrid.waitForExist();
+            const column = page.getElement(".column-header*=First Name", datagrid);
+            const icon = page.getElement("svg", column);
+            column.click();
+            expect(icon.getAttribute("data-icon")).toContain("long-arrow-alt-up");
```

Comment:
Here you are asserting that the sort icon changes, perhaps asserting the data is sorted too is a good idea? I'd like to see:

Arrange: check that data is in correct initial order
Act: click sort button
Assert: check icon and data order is correct

Reply:
Yeah, good idea! I will do that!

Reply:
Added expect for the data.

Reply:
But in that case I'm first checking the data without click to sort. And after that I'm clicking to do a sort asc. So, It's correct doesn't to change the data. Here we have this data being checked: 10 , 11 and 12.